gap/isomorph.gi

```diff
@@ -0,0 +1,556 @@
+#############################################################################
+##
+#W  isomorph.gi
+#Y  Copyright (C) 2014-17                                James D. Mitchell
+##                                                          Wilf A. Wilson
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+# This file contains methods using bliss or nauty, for computing isomorphisms,
+# automorphisms, and canonical labellings of digraphs.
+
+InstallGlobalFunction(DigraphsUseBliss,
+function()
+  # Just do nothing if NautyTracesInterface is not available.
+  if DIGRAPHS_NautyAvailable then
+    Info(InfoWarning,
```

Comment:
Why are you using `InfoWarning` here (and elsewhere) rather than `InfoDigraphs`?